src/Grids/conformal_cubed_sphere_face_grid.jl

```diff
@@ -157,8 +157,8 @@ end
 
 function offset_data(data, Hx, Hy)
```

Comment:
There's another `offset_data` function elsewhere right?

Reply:
True. https://github.com/CliMA/Oceananigans.jl/blob/c7c9a3b41452a7f3934ba2903faa7df149c5d794/src/Fields/new_data.jl#L33

Not sure what I wrote this. Will fix in PR #1522.